setup.py

```diff
@@ -38,13 +39,13 @@ def run(self):
     author="Sterling Baldwin, Tom Vo, Chengzhu (Jill) Zhang, Anthony Bartoletti",
     author_email="vo13@llnl.gov, zhang40@llnl.gov",
     description="Transform E3SM model data output into cmip6 compatable data "
-                "using the Climate Model Output Rewriter.",
-    entry_points={'console_scripts':
-                  ['e3sm_to_cmip = e3sm_to_cmip.__main__:main']},
-    packages=['e3sm_to_cmip', 'e3sm_to_cmip.cmor_handlers'],
```

Comment:
I updated the `packages` value in `setup` because it was breaking. The standard convention based on what I found is to use `find_packages()` instead of statically defining the included modules.